core/schema/container.go

```diff
@@ -670,12 +701,16 @@ func (s *containerSchema) rootfs(ctx context.Context, parent *core.Container, ar
 
 type containerExecArgs struct {
 	core.ContainerExecOpts
+
+	// If the container has an entrypoint, ignore it for this exec rather than
+	// calling it with args
+	SkipEntrypoint *bool `default:"true"`
 }
 
 func (s *containerSchema) withExec(ctx context.Context, parent *core.Container, args containerExecArgs) (*core.Container, error) {
-	if args.ContainerExecOpts.SkipEntrypoint != nil {
+	if args.SkipEntrypoint != nil {
 		slog.Warn("The 'skipEntrypoint' argument is deprecated. Use 'useEntrypoint' instead.")
-		if !args.ContainerExecOpts.UseEntrypoint && !*args.ContainerExecOpts.SkipEntrypoint {
+		if !args.ContainerExecOpts.UseEntrypoint && !*args.SkipEntrypoint {
 			args.ContainerExecOpts.UseEntrypoint = true
 		}
 	}
```

Comment:
This is interesting. So this function serves both v0.12 and v0.13, it's just that in v0.13 `SkipEntrypoint` is always `nil`, right?

Reply:
Yup, exactly :tada: I'd like to do some more cleanup around here, I think some of the changes to multiple schema versions are hard to track, but it works for now :tada: